docs/admin/backup.rst

```diff
@@ -173,37 +172,36 @@ Database
 
 The actual storage location depends on your database setup.
```

Comment:
Where?

Reply:
Probably this can be removed. The intention here was to tell that the database is stored in a different location or might be stored on another server.

Reply:
I kept it, it’s OK there.